applications/SeismicApplication/custom_conditions/fiber_beam_column_section.hpp

```diff
@@ -0,0 +1,304 @@
+//  KRATOS  ____       _               _
+//         / ___|  ___(_)___ _ __ ___ (_) ___
+//         \___ \ / _ \ / __| '_ ` _ \| |/ __|
+//          ___) |  __/ \__ \ | | | | | | (__
+//         |____/ \___|_|___/_| |_| |_|_|\___|
+//
+//  License:     BSD License
+//  license:     structural_mechanics_application/license.txt
+//
+//  Main authors: Mahmoud Zidan
+//    Co authors: Long Chen
+//
+
+#if !defined(KRATOS_FIBER_BEAM_COLUMN_SECTION_H_INCLUDED )
+#define  KRATOS_FIBER_BEAM_COLUMN_SECTION_H_INCLUDED
+
+// System includes
+
+
+// External includes
+
+
+// Project includes
+#include "includes/define.h"
+#include "includes/variables.h"
+#include "includes/element.h"
+#include "custom_conditions/fiber_beam_column_uniaxial_fiber.hpp"
+
+namespace Kratos
+{
+
+///@}
+///@name Kratos Globals
+///@{
+
+///@}
+///@name Type Definitions
+///@{
+
+///@}
+///@name  Enum's
+///@{
+
+///@}
+///@name  Functions
+///@{
+
+///@}
+///@name  Kratos Classes
+///@{
+
+/**
+ * @class FiberBeamColumnSection
+ *
+ * @brief A 3D section for fiber beam-column element
+ * @details
+ *
+ * @author Mahmoud Zidan
+ */
+
+class KRATOS_API(SEISMIC_APPLICATION) FiberBeamColumnSection
```

Comment:
If this is not a condition should be ain a different folder, not in custom_conditions